src/components/layout/NavigationBar.js

```diff
@@ -3,6 +3,7 @@ import { Nav, Navbar, NavDropdown, Container } from "react-bootstrap";
 import { Link } from "react-router-dom";
```

Comment:
since we are using Image component from react-bootstrap we  need to add `Image`

`import { Nav, Navbar, NavDropdown, Container, Image } from "react-bootstrap";`

Reply:
Yes